marshmallow_utils/fields/babel.py

```diff
@@ -209,4 +210,7 @@ def gettext_from_dict(catalog, locale, default_locale):
         return catalog[catalog_key]
     # If not, use default locale (must be defined it is defined)
     # "en" is set as fallback lng.
-    return catalog.get(str(default_locale), "en")
+    try:
+        return catalog[str(default_locale)]
+    except KeyError:
+        return catalog["en"]
```

Comment:
Shouldn't this be:

```python
# A: might return None 
return catalog.get(str(default_locale), default=catalog.get("en"))

# B: variation of A
return catalog.get(str(default_locale)) or catalog.get("en")

# C: take the first available value
return catalog.get(str(default_locale)) or next(iter(catalog.values()), None)
```

See #88 for why, since right now we basically get a 500 bubbling up when there's no `en` key in the dictionary.